### PR TITLE
Adresses CSP violation [Fixes #6684]

### LINKED ISSUE #6684: CSP violation on /about/forums/
URL: https://www.mozilla.org/en-US/about/forums/

CSP error: 
```
Content Security Policy: The page's settings blocked the loading of a resource at https://www.google.com/logos/Logo_25wht.gif ("img-src").
```

HTML reference: https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/forums/forums.html#L42

Resolution: we should either remove this image reference, or host it ourselves.

### PR DESCRIPTION
**Pull request by Errolyn**
## Description
- added SVG to repo
- made sure logo link takes up appropriate space

## Issue / Bugzilla link
#6684

### CODE REVIEW

**Comment by Errolyn** [comment]
@alexgibson I have renamed the branch so you should be able to pull it down and test it 🐕

File: bedrock/mozorg/templates/mozorg/about/forums/forums.html [hunk 1]
```diff
@@ -36,16 +36,15 @@ <h1><span class="highlight">{{ self.page_title() }}</span></h1>
   {{ _('Mozilla newsgroups are sponsored by <a href="%s">Giganews Newsgroups</a>, to whom we are very grateful.')|format('https://www.giganews.com/') }}
   <a href="https://www.giganews.com/"><img src="{{ static('img/mozorg/about/giganews.png') }}" alt="" height="25" width="89" /></a>
   </p>
-
+  <a href="https://www.google.com/" ><img
+    src="{{ static('img/logos/google/logo_google.svg') }}"
+    width="75" height="32" alt="Google" /></a>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/forums/forums.html]: The new image ratio is a little different, so the logo looks a little squished. 

image


Setting `height="25"` seems to be about right.

File: bedrock/mozorg/templates/mozorg/about/forums/forums.html [hunk 2]
```diff
@@ -36,16 +36,15 @@ <h1><span class="highlight">{{ self.page_title() }}</span></h1>
   {{ _('Mozilla newsgroups are sponsored by <a href="%s">Giganews Newsgroups</a>, to whom we are very grateful.')|format('https://www.giganews.com/') }}
   <a href="https://www.giganews.com/"><img src="{{ static('img/mozorg/about/giganews.png') }}" alt="" height="25" width="89" /></a>
   </p>
-
+  <a href="https://www.google.com/" ><img
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/forums/forums.html]: NIt: also suggest putting this `` element on a single line vs splitting over multiple (this is a really old template)

**Comment by Errolyn** [comment]
ok so I have messed up this branch 😿 I will get it figured but I need to unwind stuff before this is reviewed again.

**Comment by Errolyn** [comment]
> ok so I have messed up this branch 😿 I will get it figured but I need to unwind stuff before this is reviewed again.

Ok I have fixed the branch issues, I am now going to try to rebase and push my changes.

**Comment by Errolyn** [comment]
There were some ups and downs, but we made it there... without having to start a PR from scratch a third time. 😅 ready for another review.

**Pull request merged**